README.md

```diff
@@ -703,10 +703,28 @@ Default: `true`
 
 #### omitSkippedTests ####
 
-Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCuucumberStyleTestReports is not set to `true`
+Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCucumberStyleTestReports is not set to `true`
 
 Default: `true`
 
+#### pylintDisabledChecker ####
```

Comment:
S (Substantive): would be better if we took the underlying library name out and just called it linting.  I realize we've made this mistake elsewhere, but would prefer not to add on more to that now.

Reply:
Addressed, thanks Ryan for the suggestion!